### DISCUSSION TITLE
Someone is using SysErrOutOfGas to perform fake deposit

### DISCUSSION THREAD

**Discussion by Ljzn** [filecoin-project/lotus#9076] [General Q&A]
https://filfox.info/en/address/f13uxocsayb6dhrmzzkfsefxobrtqmtq4gtkl7agq almost every transactions from this address are failed with "SysErrOutOfGas". And it's continue send more txs.

**Comment by arajasek** [comment]
I would guess someone has bad tooling for their gas estimation, I wouldn't call this a "fake deposit". Kinda looks like they're just wasting their own FIL...
